### PR TITLE
5 + 6 individual article and associated comments

### PR DESCRIPTION
**Pull request by Arron-B**

### CODE REVIEW

File: src/components/Article.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+							return (
+								<Card key={`comment${comment.comment_id}`}>
+									<Card.Subtitle>{comment.author}</Card.Subtitle>
+									<Card.Body>{comment.body}</Card.Body>
+								</Card>
+							);
+						})}
+					</section>
+				</DropdownButton>
+			</Card>
```
Comment (Poonam-raj) [inline src/components/Article.jsx:75]: I'm not convinced a 'card' tag is the best choice for a single article. Really this is the entire purpose of the page and feels more appropriate to use a 'artlcle' tag or something more concrete than a card which I'd expect for a list situation. But this is more a personal choice, change it if you want.

File: src/components/Article.jsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+						{comments.map((comment) => {
+							return (
+								<Card key={`comment${comment.comment_id}`}>
+									<Card.Subtitle>{comment.author}</Card.Subtitle>
+									<Card.Body>{comment.body}</Card.Body>
+								</Card>
+							);
+						})}
+					</section>
+				</DropdownButton>
```
Comment (Poonam-raj) [inline src/components/Article.jsx:74]: displaying the comments as a drop down really doesn't work from a UX point of view. Please rethink this and display it inline on the page in a more concrete way.

File: src/components/Article.jsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+	if (article) {
+		return (
+			<Card>
+				<Card.Title>{article.title}</Card.Title>
+				<Card.Subtitle>Posted: {article.created_at}</Card.Subtitle>
+				<Card.Subtitle>{capitaliseFirstLetter(article.topic)}</Card.Subtitle>
+				<Card.Subtitle>Author: {article.author}</Card.Subtitle>
+				<Card.Img
+					variant="top"
+					src={article.article_img_url}
```
Comment (Poonam-raj) [inline src/components/Article.jsx:42]: please add some alt text to the image. this can be dynamic

File: src/App.jsx [hunk 4]
```diff
@@ -16,6 +17,10 @@ function App() {
 					path="/"
 					element={<Home />}
 				/>
+				<Route
+					path="/articles/:article_id"
+					element={<Article />}
+				/>
 			</Routes>
```
Comment (Poonam-raj) [inline src/App.jsx:24]: it's fine having the route for articles/:article_id but one massive issue is that i have no way of getting to the article via the articles list which is the most logical way for me to do it

File: src/components/Article.jsx [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+} from "react-bootstrap";
+import { capitaliseFirstLetter } from "../utils/utils";
+
+function Article() {
+	const { article_id } = useParams();
+
+	const [article, setArticle] = useState(false);
+	const [votes, setVotes] = useState(0);
+	const [comments, setComments] = useState([]);
+
```
Comment (Poonam-raj) [inline src/components/Article.jsx:19]: would make sense to introduce loading state at this point, it's really useful throughout the app when i throttle my internet to know the app is still working just taking some time.

Comment (Arron-B) [reply src/components/Article.jsx:19]: Loading is already implemented. It is just dependent on the article state, rather than an isLoading state.

**Pull request merged**